scripts/tools/common.yml

```diff
@@ -125,6 +125,26 @@ etors:
       desc: $t_metric_calculate_exp_desc_t
       version: "1.7"
       value: "0x00010002"
+    - name: METRIC_PROGRAMMABLE_EXP_PROPERTIES
+      desc: $t_metric_programmable_exp_properties_t
+      version: "1.8"
+      value: "0x00010003"
+    - name: METRIC_PROGRAMMABLE_PARAM_INFO_EXP
+      desc: $t_metric_programmable_param_info_exp_t
+      version: "1.8"
+      value: "0x00010004"
+    - name: METRIC_PROGRAMMABLE_PARAM_VALUE_EXP
+      desc: $t_metric_programmable_param_value_exp_t
+      version: "1.8"
+      value: "0x00010005"
+    - name: METRIC_PROGRAMMABLE_PARAM_VALUE_INFO_EXP
+      desc: $t_metric_programmable_param_value_info_exp_t
+      version: "1.8"
+      value: "0x00010006"
+    - name: VALUE_INFO_EXP
```

Comment:
why this one is not prefixed with METRIC_?

Reply:
Existing "Value" related structures / enums does not seems to be prefixed with metrics.
For example: t_value_type_t

So thought of having the value_info_x structure without a metric prefix.